mpas_analysis/shared/generalized_reader/generalized_reader.py

```diff
@@ -91,6 +92,11 @@ def open_multifile_dataset(fileNames, calendar, config,
         If present, the first and last dates to be used in the data set.  The
         time variable is sliced to only include dates within this range.
 
+    chunking : None, True, dict, optional
```

Comment:
If I understand, you are actually supplying an integer, which is not one of the possibilities you give.  Could you explain here what the behavior is when an integer is supplied?